### ISSUE TITLE
Add a UIV/UMV Wireless Laser option

### ISSUE THREAD

**Issue by redspah** [GTNewHorizons/GT-New-Horizons-Modpack#23853]
### Your GTNH Discord Username

redspah

### Your Pack Version

2.9 Daily

### Your Proposal

Add a wireless laser pipe to make setting up lasers en masse less annoying in UIV/UMV. It could double as a pre-Eternity use for Quark-Gluon Plasma, which currently has no meaningful uses in UMV.

### Your Goal

Wireless Multi-Amp hatches are a great addition, but they cap out at UMV 64A, which isn't enough for many machines that allow lasers by UMV standards. (ex. DTPF, AAL) This forces the player to plop down a wireless LSC and connect a bunch of lasers from it to each individual machine, which gets very cumbersome very fast.

A wireless laser pipe would make bulk setup easier without eliminating the intended expensive components, i.e. laser hatches. Making it cost QGP would give players incentive to setup Heliofusion Exoticiser when they first unlock it, because currently there's no point to doing so.

To maintain the flavor of laser energy, wireless lasers could have similar positioning requirements as Miniature Wormhole Generators (which they are effectively singleblock versions of), where input/output laser pipes are on the opposite sides + must be of the same colour. If lossless transfer is deemed too OP, they could also have a modest (up to 0.5%) energy loss, but to avoid annoying powerfails, that energy loss would be instead subtracted directly from the wireless network. Crossdim functionality would probably not be OP by UMV standards either.

Wireless lasers would be connected with data sticks.

Example recipe: 
16 Dimensional Injection Casing
8 Laser Vacuum Mirror
1 Superdense Chromatic Glass Plate
2 UIV Emitter
2 UIV Sensor
64 Fine SpaceTime Wire 
2 UMV Circuit
4 AE2 Singularity
1 Energized Tesseract
16kL Grade 8 Water
1440L Radox Polymer
64kL Quark-Gluon Plasma
---> 2 **Compact Photonic Wormhole Pipe**
10s @ UMV @ Assembly Line

### Your Vision

Less tedious nonsense with setting up DTPFs en masse, extra use to one of the new 2.8 materials

### Final Checklist

- [x] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying *I like this feature please reconsider adding it* will prompt us to investigate and reopen it once we confirm your report.
- [x] I believe there is nothing similar in the pack already, or the existing solution isn't good enough.
- [x] I understand this change request may not attract enough attention and thus not be implemented.
- [x] I understand this change request may be rejected due to other community members thinking it's inappropriate.
- [x] I believe this feature would make the pack better.

**Comment by UltraProdigy** [comment]
These exist already, they're just unlocked in UXV. Are you proposing a downtier or something different and I'm misunderstanding?

Edit: Oh I see what you mean. Still individual sources but wirelessly linked instead of just wireless receivers like in UXV

**Comment by redspah** [comment]
@UltraProdigy Yes, that's what I mean, normal laser hatches that are wirelessly linked

**Comment by UltraProdigy** [comment]
My only trouble with that idea is that it almost makes the UXV wireless lasers pointless. If in UMV I already setup wireless arrays, and I can do so without the (I believe Universium) cost of the UXV wireless then I have no incentive to create the UXV wireless barring maybe AALs. Unless the power was limited- but in that case you would just end up going back to the LSC + laser combo.

**Comment by redspah** [comment]
How about having the energy penalty scale with the EU/t? Though I'm not sure off the top of my head how to balance it in a way that makes it worthwhile to use while still encouraging UXV wireless lasers for actual *really* high power application. 

Something like `%_loss = eut / 1e12` could be small enough for to make the wireless ones still worth using for eg. DTPFs and AALs in UMV, while being prohibitively expensive for anything above UMV 1kA.

**Comment by Nana-Sakura** [comment]
> These exist already, they're just unlocked in UXV. Are you proposing a downtier or something different and I'm misunderstanding?
> 
> Edit: Oh I see what you mean. Still individual sources but wirelessly linked instead of just wireless receivers like in UXV

Well I think we can just make the wireless laser hatch, but that should be made with a lapotron capacitor (tiered), LSC controller, 41 UV glasses, and requiring the laser source and the laser target to make
